Torch map fixes

Kasuobes opened the pull request:
Fixes #15842
Fixes #15844
Fixes #15845
Fixes #15880
Fixes #15913
Fixes #15917 

Removed mapped in magboots with emergency o2 tanks because every SSU come with boots now.
Replaced portable generators on shuttles with less dangerous ones.

Kasuobes committed:
Map fixes

Raptor1628 commented:
Did you replace the generator fuel as well?

Kasuobes commented:
Generators start with phoron sheets loaded, yes.

Techhead0 reviewed (approved):
Looks good to me.

Raptor1628 commented:
Yes, but there are crates with spare fuel for the generators, have you replaced the uranium with phoron?

Kasuobes commented:
Replaced uranium fuel with phoron, also added in more fixes for telecomms getting knocked out by storms, away z APC warnings showing up, power not flowing properly to 4th deck, awkward gas tank placements on shuttles, one section of pipe from the Aquila that keep getting GC'd on initialization.

mwerezak commented:
The phoron generators go through fuel significantly faster, did you give them more fuel to reflect that?

Kasuobes commented:
Yes, the fuel crates now contain enough phoron that mining would never need to go get any.

Raptor1628 commented:
Rarity of phoron is part of the reason I used uranium.

Kasuobes commented:
Phoron is twice as common as uranium when it comes to asteroid generation, not to mention the many phoron gas cans lying around and their many applications.

Lore should probably be revised so phoron isn't the super rare mineral/gas as it was before, which was frankly ridiculous to begin with seeing how common it is.
